homeassistant/components/history.py

```diff
@@ -182,7 +178,7 @@ def get(self, request, date=None):
         one_day = timedelta(days=1)
 
         if date:
-            start_time = dt_util.as_utc(dt_util.start_of_local_day(date))
+            start_time = dt_util.as_utc(date)
```

Comment:
The extra_url needs updating here too and the parameter should not be called `date`